docs/docs/guide/glossary.md

```diff
@@ -51,7 +51,7 @@ Represents an agent or actor in the system.
 
 ### Vault
 
-ECHO data and HALO identities are physically stored in browser storage on the [HALO](./platform/halo) application's domain, not the consuming application's. **This enables data privacy and gives end-users control over their data**. Read more about [HALO vault topology](./platform/#local-vault-topology).
+ECHO data and HALO identities are physically stored in browser storage on the application's domain.
```

Comment:
I'd probably just remove this term from the glossary altogether in the interim...

Reply:
avoids having to redefine it later